app/views/paye/PayeMultipleYearsView.scala.html

```diff
@@ -36,7 +36,7 @@
 
             @inputRadioGroupView(
             atsYearForm(AtsForms.yearChoice),
-            years.map(year => year.toString -> messages("ats.select_tax_year.label", year.toString, s"${year + 1}")),
+            years.map(year => year.toString -> messages("ats.select_tax_year.label", (year-1).toString, s"${year}")),
```

Comment:
Does this show the correct year now? Just aware that this may be an unwanted change from merging from master

Reply:
Yes it shows the correct year now. Since now we are using 2020 as CY so taxYearFrom should be CY-1 and  taxYearTo should be CY.